smt_switch/src/solvers/BoolectorSolver.py

```diff
@@ -92,25 +93,26 @@ def set_nonstandard_option(self, optionstr, value):
     def declare_const(self, name, sort):
```

Comment:
In SMT-LIB v2.6 declare-const is used to define nullary function symbols. What about function symbols with arity > 0?